src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -30,18 +30,17 @@ public TorpedoStore(int numberOfTorpedos){
 
   public boolean fire(int numberOfTorpedos){
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+      throw new IllegalArgumentException("numberOfTorpedos");
     }
 
     boolean success = false;
 
     // simulate random overheating of the launcher bay which prevents firing
-    Random generator = new Random();
     double r = generator.nextDouble();
 
     if (r >= FAILURE_RATE) {
       // successful firing
-      this.torpedoCount =- numberOfTorpedos;
+      this.torpedoCount -= numberOfTorpedos;
```

Comment:
Jó megoldás, de ezt a sort írd át légyszi erre:
this.torpedoCount = this.torpedoCount - numberOfTorpedos;

Reply:
Köszi! Javítva az [5e1d052](https://github.com/octolith/ivt-lab/pull/3/commits/5e1d05223942bef928d22d1d37d8ec6ca908cd5c) commitban.